[2024-05-10T07:04:10.917+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-10T07:04:10.982+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2024-04-27T00:00:00+00:00 [queued]>
[2024-05-10T07:04:11.011+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2024-04-27T00:00:00+00:00 [queued]>
[2024-05-10T07:04:11.024+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-10T07:04:11.060+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-04-27 00:00:00+00:00
[2024-05-10T07:04:11.071+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=421) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-10T07:04:11.088+0000] {standard_task_runner.py:63} INFO - Started process 469 to run task
[2024-05-10T07:04:11.072+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'insert_into_table', 'scheduled__2024-04-27T00:00:00+00:00', '--job-id', '5573', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpgwleje29']
[2024-05-10T07:04:11.091+0000] {standard_task_runner.py:91} INFO - Job 5573: Subtask insert_into_table
[2024-05-10T07:04:11.279+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2024-04-27T00:00:00+00:00 [running]> on host ba9ec80fe13a
[2024-05-10T07:04:11.470+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-04-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-27T00:00:00+00:00'
[2024-05-10T07:04:11.476+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-10T07:04:11.513+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-04-27', 'dag_with_postgres_operator_v01')
        
[2024-05-10T07:04:11.541+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-05-10T07:04:11.568+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-05-10T07:04:11.596+0000] {sql.py:457} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-04-27', 'dag_with_postgres_operator_v01')
        , parameters: None
[2024-05-10T07:04:11.602+0000] {sql.py:466} INFO - Rows affected: 1
[2024-05-10T07:04:11.608+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-05-10T07:04:11.624+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-10T07:04:11.650+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=insert_into_table, execution_date=20240427T000000, start_date=20240510T070410, end_date=20240510T070411
[2024-05-10T07:04:11.692+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-10T07:04:11.718+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-10T07:04:11.729+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
